app/models/service_template_transformation_plan_task.rb

```diff
@@ -71,6 +71,15 @@ def preflight_check
     destination_cluster
     virtv2v_disks
     network_mappings
+
+    begin
+      raise unless source.host.verify_credentials?
```

Comment:
Not sure this is a good idea to do sync.  @agrare ?

Reply:
Yeah you probably want to use `host.authentication_status_ok?` which goes to the database instead of `verify_credentials` which actually goes out and does an API call to the host.